plugins/template-validator.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const validator = require('screwdriver-template-validator');
+
+exports.register = (server, options, next) => {
+    server.route({
+        method: 'POST',
+        path: '/validator/template',
```

Comment:
I think `/validator/template` makes sense. 
Kinda like how we have `sync/webhooks` and `sync/pullrequests`